packages/fxa-auth-server/scripts/must-reset/index.js

```diff
@@ -0,0 +1,58 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+'use strict';
+
+module.exports = async function main(keys, dbFunction) {
+  const butil = require('../../lib/crypto/butil');
+  const config = require('../../config').getProperties();
+  const crypto = require('crypto');
+  const log = require('../../lib/log')({});
+  const P = require('../../lib/promise');
+  const Token = require('../../lib/tokens')(log, config);
+  const AuthDB = require('../../lib/db')(config, log, Token);
+  const oauth = require('../../lib/oauth/db');
```

Comment:
We typically put these above the export. Is there some other reason for putting the imports in the function?

Reply:
I just implemented it the same way as dump-user.js

Reply:
Sigh, yup.